src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs

```diff
@@ -1630,6 +1654,63 @@ private void MakeSeparatorList(ReadOnlySpan<char> separators, ref ValueListBuild
             }
         }
 
+        private void MakeSeparatorListVectorized(ref ValueListBuilder<int> sepListBuilder, char c, char? c2 = null, char? c3 = null)
+        {
+            // Constant that defines indices of characters within an AVX-Register
+            const ulong indicesConstant = 0xFEDCBA9876543210;
```

Comment:
I'd move this const near to it's usage, as the code is easier to read when things are closely together.

Reply:
Done https://github.com/dotnet/runtime/pull/38001/commits/2b0b8f8d63b8a3e5af5d2d3e2160c17efc245561